src/xdocs/checks/design/innertypelast.xml

```diff
@@ -17,13 +17,17 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p>
+        <p id="Example1-config">
           To configure the check:
         </p>
         <source>
-&lt;module name="InnerTypeLast"/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;InnerTypeLast&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>Example:</p>
+        <p id="Example1-code">Example:</p>
         <source>
 class Test {
     private String s; // OK
```

Comment:
So, this is generated with no diff?

Reply:
Yes, because it is a copy-paste. With the delimiters around it. We expect no diff here.

Reply:
To be clear:

"copy-paste" == human activity

"generated" == computer activity

Proof that generation works with no diff:
```
➜  checkstyle git:(pull-13482) ✗ git diff -u | cat
diff --git a/src/xdocs/checks/design/innertypelast.xml b/src/xdocs/checks/design/innertypelast.xml
index 80ef9c03f..851e0e645 100644
--- a/src/xdocs/checks/design/innertypelast.xml
+++ b/src/xdocs/checks/design/innertypelast.xml
@@ -29,7 +29,7 @@
         </source>
         <p id="Example1-code">Example:</p>
         <source>
-class Test {
+class Test1234 {
     private String s; // OK
     class InnerTest1 {}
     public void test() {} // violation, method should be declared before inner types.
➜  checkstyle git:(pull-13482) ✗ mvn clean verify
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 1 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 1 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[INFO] 
[INFO] ------------------< com.puppycrawl.tools:checkstyle >-------------------
[INFO] Building checkstyle 10.12.2-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.3.1:clean (default-clean) @ checkstyle ---
[INFO] Deleting /home/nick/IdeaProjects/checkstyle/target
...
[INFO] Loaded JSON from /home/nick/IdeaProjects/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/sariflogger/ExpectedSarifLoggerSingleWarning.sarif
[INFO] Successfully validated JSON from /home/nick/IdeaProjects/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/sariflogger/ExpectedSarifLoggerSingleWarning.sarif against /home/nick/IdeaProjects/checkstyle/target/plexus-resources1263236444tmp
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  06:15 min
[INFO] Finished at: 2023-07-29T09:55:01-04:00
[INFO] ------------------------------------------------------------------------
➜  checkstyle git:(pull-13482) git diff -u | cat
➜  checkstyle git:(pull-13482) 
```

Awesome!